server/lib/adapters/mssql.js

```diff
@@ -218,7 +219,7 @@ export function streamQuery(client, query, options = {}) {
     const confoptions = _.merge(client.config.options, options);
     const conf = _.cloneDeep(client.config);
     conf.options = confoptions;
-    const conn = new tedious.Connection(conf);
+    const conn = client;//new tedious.Connection(conf);
```

Comment:
This is the primary fix. We had already created a connection through the "openConnection" method. Don't need to do it again, just use the one passed in. closeConnection is called on it in syncagent

- Streaming

Reply:
I would remove the commented out //new tedious.Connection(conf); to avoid future confusion